app/static/tests/unit/components/options/parameterSetView.test.ts

```diff
@@ -283,10 +284,10 @@ describe("ParameterSetView", () => {
         expect(displayNameInput.element.value).toBe("Set 1");
         expect(displayNameText.isVisible()).toBe(false);
 
-        // 1 for onMount, 1 for ref saveButton update, 1 for editIcon click
-        expect(mockTooltipDirective).toHaveBeenCalledTimes(15);
-        expect(mockTooltipDirective.mock.calls[11][0]).toBe(saveIcon.element);
-        expect(mockTooltipDirective.mock.calls[11][1].value).toBe("Save Parameter Set Name");
+        // 4 on mount + 1 input on mount + 4 after update
```

Comment:
nice comment rot on the lhs (1 + 1 + 1 = 15???)

Reply:
haha yh XD i think it was supposed to be one batch so 5 + 5 + 5 but yh bad comment from me in the past haha